src/app/templates/login.ftl

```diff
@@ -4,18 +4,29 @@
     <title>${title}</title>
     <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
     <link rel="stylesheet" href="/public/css/style.css"/>
+    <script>
```

Comment:
Unfortunately this bit of javascript isn't tested, but I didn't think it was worth converting into a vue component just for that.. What do you think?

Reply:
Totally agree, this seems fine to me!